dev/com.ibm.ws.ejbcontainer.mdb.ra_fat/test-applications/MsgEndpointEJB.jar/src/com/ibm/ws/ejbcontainer/fat/msgendpoint/ejb/MDBTimedCMTBean.java

```diff
@@ -75,6 +74,8 @@
 
     public MDBTimedCMTBean() {}
 
+    private static boolean svCancelTimer = false;
```

Comment:
Perhaps could add volatile to this to ensure the timer thread sees the change in state immediately after being set to true